pikaur/pprint.py

```diff
@@ -115,16 +94,8 @@ def pretty_format(pkg_update):
             pkg_len += len(pkg_update.Repository) + 1
         return ' {pkg_name}{spacing} {current_version}{spacing2} -> {new_version}{verbose}'.format(
             pkg_name=pkg_name,
-            current_version=_color_line(common_version, version_color) +
```

Comment:
i think you should revert this change chunk and so remove the method `diff_print` from PackageVersion class

Reply:
and this one still not addressed

Reply:
How is this not addressed?

Reply:
my bad, didn't pulled the local copy, now this is fine